WcaOnRails/spec/controllers/delegate_reports_controller_spec.rb

```diff
@@ -54,7 +54,7 @@
       expect(comp.delegate_report.remarks).to eq "My new remarks"
     end
 
-    it "can edit report before comp is over, but cannot post report" do
+    it "can edit report before comp is over, and can post report" do
```

Comment:
Good call on adding the expect, it makes the test easier to read.

I'm not sure about "probably over" here, what do we fudge by one day in the code?